app/models/spree/paga_transaction.rb

```diff
@@ -78,7 +78,7 @@ def update_payment_source
   end
 
   def update_transaction_status
-    paga_notification = Spree::PagaNotification.where(:transaction_id => self.transaction_id).first
+    paga_notification = Spree::PagaNotification.where(transaction_id: self.transaction_id).first
```

Comment:
@nimish13 DRY not followed `update_transaction_status`.
Similar method in `Spree::PagaNotification`
Also use `find_by` here.